stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/BackdropComponent.kt

```diff
@@ -83,88 +92,99 @@ fun Backdrop(
     BackdropScaffold(
         modifier = modifier,
         appBar = {
-            Toolbar(
-                settingsUiState.selectedTransactionItem.label,
-                settingsUiState.fromFacilitiesLabel().asString(),
-                settingsUiState.deliverToLabel()?.asString(),
-                themeColor,
-                launchBottomSheet = {
-                    manageStockViewModel.onHandleBackNavigation()
-                },
-                backdropState,
-                scaffoldState,
-                syncAction,
-                settingsUiState.hasFacilitySelected(),
-                settingsUiState.hasDestinationSelected(),
-            )
+            if (selectedIndex == BottomNavigation.ANALYTICS.id) {
```

Comment:
It seems that original Backdrop and Analitycs content are not sharing any component, do you thing could be an option to create a separated AnalyticScreen with the AnalyticToolbar and AndroidView instead of checking the selectedIndex multiple times here?

Reply:
Thanks for the feedback! I did refactor this